efekt.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <link rel="stylesheet" href="mysz.css">
+    <meta charset="UTF-8">
+    <title>Efekty po najechaniu myszą</title>
+</head>
+<body>
+
+<h1>Efekty po najechaniu myszą</h1>
+
+<h2>Kolor tekstu</h2>
+<p>
+    Aliquam non ut hic ea et quam. <a href="a"><span>Dolores esse quis totam</span></a> aut et consequatur quia hic iste commodi quis
```

Comment:
```suggestion
    Aliquam non ut hic ea et quam. <a href=""><span>Dolores esse quis totam</span></a> aut et consequatur quia hic iste commodi quis
```
`href` moze byc pusty